src/MDMMotionAnimator.h

```diff
@@ -69,73 +62,91 @@ NS_SWIFT_NAME(MotionAnimator)
  animations added from prior invocations.
 
- @param timing The timing to be used for the animation.
+ @param traits The traits to be used for the animation.
  @param layer The layer to be animated.
  @param values The values to be used in the animation. Must contain exactly two values. Supported
  UIKit types will be coerced to their Core Animation equivalent. Supported UIKit values include
  UIColor and UIBezierPath.
  @param keyPath The key path of the property to be animated.
  */
-- (void)animateWithTiming:(MDMMotionTiming)timing
-                  toLayer:(nonnull CALayer *)layer
-               withValues:(nonnull NSArray *)values
+- (void)animateWithTraits:(nonnull MDMAnimationTraits *)traits
+                  between:(nonnull NSArray *)values
+                    layer:(nonnull CALayer *)layer
                   keyPath:(nonnull MDMAnimatableKeyPath)keyPath;
 
 /**
- Adds a single animation to the layer with the given timing structure.
+ Adds a single animation to the layer with the given traits structure.
 
  If `additive` is disabled, the animation will be added to the layer with the keyPath as its key.
  In this case, multiple invocations of this function on the same key path will remove the
  animations added from prior invocations.
 
- @param timing The timing to be used for the animation.
+ @param traits The traits to be used for the animation.
  @param layer The layer to be animated.
  @param values The values to be used in the animation. Must contain exactly two values. Supported
  UIKit types will be coerced to their Core Animation equivalent. Supported UIKit values include
  UIColor and UIBezierPath.
  @param keyPath The key path of the property to be animated.
  @param completion A block object to be executed when the animation ends or is removed from the
  animation hierarchy. If the duration of the animation is 0, this block is executed immediately.
- The block is escaping and will be released once the animations have completed.
+ The block is escaping and will be released once the animations have completed. The provided
+ didComplete argument is currently always YES.
  */
-- (void)animateWithTiming:(MDMMotionTiming)timing
-                  toLayer:(nonnull CALayer *)layer
-               withValues:(nonnull NSArray *)values
+- (void)animateWithTraits:(nonnull MDMAnimationTraits *)traits
+                  between:(nonnull NSArray *)values
+                    layer:(nonnull CALayer *)layer
                   keyPath:(nonnull MDMAnimatableKeyPath)keyPath
-               completion:(nullable void(^)(void))completion;
+               completion:(nullable void(^)(BOOL didComplete))completion;
+
+/**
+ If enabled, explicitly-provided values will be reversed before animating.
+
+ This property only affects the animateWithTraits:between:... family of methods.
+
+ Disabled by default.
+ */
+@property(nonatomic, assign) BOOL shouldReverseValues;
```

Comment:
If these are supposed to be "grouped", consider bundling them into a category.

Reply:
This is the canonical way to have APIs grouped in API docs. See the attached jazzy output:

<img width="644" alt="screen shot 2017-12-13 at 3 35 39 pm" src="https://user-images.githubusercontent.com/45670/33961242-536bb02e-e01b-11e7-969b-6cb400f29d5a.png">

Reply:
Ack.